.travis.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   include:
     - stage: Static analysis (src) with phpstan
       php: 7.2
-      script: vendor/bin/phpstan analyse --level=7 src
+      script: make phpstan
```

Comment:
Please just put the executed code here, no intermediate build tools

Reply:
Would it be okay to still use `make ./phpstan.phar` to download the phar? or do you want the full wget here as well? That could cause differences in the versions used for local development and the version used on travis.

Reply:
I'd probably rather have a `composer.lock` handle that, heh